dtfs-central-api/src/v1/routes/utilisation-reports-routes.js

```diff
@@ -136,10 +136,6 @@ utilisationReportsRouter
  *       500:
  *         description: Internal Server Error
  */
-utilisationReportsRouter
-  .route('/set-status')
-  .put(
-    putUtilisationReportStatusController.putUtilisationReportStatus,
-  );
+utilisationReportsRouter.route('/set-status').put(putUtilisationReportStatusController.putUtilisationReportStatus);
```

Comment:
Does this not have an ID in it right now because we support bank ID and report period @JoshBinns2000 ?